lib/viewvc.py

```diff
@@ -4021,7 +4021,8 @@ def _fp_vclib_raw(self, left, right, fp, propname):
     def _content_lines(self, side, propname):
         f = self.request.repos.openfile(side.path_comp, side.rev, {})[0]
         try:
-            lines = f.readlines()
+            lines = [lin.decode(self.request.repos.encoding, 'surrogateescape')
+                     for lin in f.readlines()]
```

Comment:
is `lin` here meant to be `line`?

Reply:
Aaah, I had used '`l`' for the variable name at first, but flask8 blames "E741 ambiguous variable name 'l'".  Then I renamed it into '`lin`'. 

However, yes,  it is no mean that reduce only one character from full spell.  It's a same mistake of creat(2) :)

Reply:
Corrected in [1cad649](https://github.com/viewvc/viewvc/pull/298/commits/1cad6499cf480b950351dde81ffb9ae94cfd1cde)